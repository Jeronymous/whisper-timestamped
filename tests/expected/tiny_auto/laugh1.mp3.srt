1
00:00:00,060 --> 00:00:01,880
You can't do it, you can't do it.

