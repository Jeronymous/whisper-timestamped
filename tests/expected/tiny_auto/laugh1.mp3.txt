You can't do it, you can't do it.
